### ISSUE TITLE
getting system error 1219 mapping network drive 

### ISSUE THREAD

**Issue by sharicov** [jbara2002/windows-defender-remover#50]
After applying remove Windows Defender and connected Windows Security Components. (first option) unable to connect my nas drive (samba 4 running on arm64 linux).
Getting following error:

```
System error 1219 has occurred.

Multiple connections to a server or shared resource by the same user, using more than one user name, are not allowed. Disconnect all previous connections to the server or shared resource and try again.
```

Same network drive works well on other windows clients.
Also was working on this specific PC before running windows-defender-remover

Windows version: 
22H2

Samba version (on linux server): Samba version 4.13.13-Debian
Protocol (with other clients): SMB3_11

**Comment by jbara2002** [comment]
so if not, i'll check in my part. but i don't have any network drives to reproduce that, until i do , i'll send hotfixes and more in this issue.

**Comment by sharicov** [comment]
Thanks a lot, really appreciate your support, you can use a Linux VM as a SAMBA server for the testing.

**Comment by NudeNULL** [comment]
Hi, having exactly the same issue, can't connect a NAS shared drive

**Comment by NudeNULL** [comment]
First, I tried to use the latest version and it didn't work at all, using the H option, then I downloaded 12.3.1, which alwasy worked for me in the past, and the defender was disabled, but I think the first attempt of using the latest version broke for me the shared drive connection, will try using your steps to undo the registry changes.

**Comment by NudeNULL** [comment]
Actually deleting this value helped
[HKLM\SYSTEM\CurrentControlSet\Control\Lsa]
LmCompatibilityLevel=dword:00000002

**Comment by sharicov** [comment]
> Actually deleting this value helped [HKLM\SYSTEM\CurrentControlSet\Control\Lsa] LmCompatibilityLevel=dword:00000002

Helped indeed, thanks a lot!

**Comment by jbara2002** [comment]
the value will not be added in next version of the app.

**Comment by sharicov** [comment]
Thanks a lot! Much appreciated!

**Comment by jbara2002** [comment]
fixed in 12.4.6
